src/controllers/task.ts

```diff
@@ -38,5 +39,15 @@ const addTask = (req: Request, res: Response, next: NextFunction) => {
     })
     .catch(() => next(new CustomError(500, 'server error')))
 };
+const deleteTaskById = (req: TokenRequest, res: Response, next: NextFunction) => {
+  const { taskId } = req.params;
+  deleteTaskByIdQuery(+taskId)
+    .then(() => {
+      res.status(204).json({

```

Comment:
204 status means that is no content returned, so we don't have any response here.

A [204](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204) (No Content) status code if the action has been enacted and no further information is to be supplied.
A [200](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200) (OK) status code if the action has been enacted and the response message includes a representation describing the status.

 read more [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE)